integration-tests/test_ftp.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import unicode_literals
+
+import logging
+import unittest
+
+from smart_open import open
+
+class FTPTest(unittest.TestCase):
+    
+    def test_nonbinary(self):
+        file_contents = "Test Test \n new test \n another tests"
+        appended_content1 = "Added \n to end"
+
+        with open("ftp://user:123@localhost:21/home/user/dir/file", "w") as f:
+            f.write(file_contents)
+            f.close()
+
+        with open("ftp://user:123@localhost:21/home/user/dir/file", "r") as f:
+            read_contents = f.read()
+            assert read_contents == file_contents
+            f.close()
```

Comment:
Why do you need to close here? The context manager does that for you.

Reply:
Good catch. I have removed it.